x-pack/platform/plugins/shared/ml/public/application/components/job_messages/job_messages.tsx

```diff
@@ -66,23 +66,24 @@ export const JobMessages: FC<JobMessagesProps> = ({
           ''
         ),
         render: (message: JobMessage) => <JobIcon message={message} />,
-        width: `${euiTheme.size.l}`,
+        width: `${euiTheme.size.xl}`,
+        align: 'center',
```

Comment:
I think this will need some rework as these column widths don't work so well inside the expanded rows in the jobs lists:

<img width="2094" alt="Screenshot 2024-12-18 at 15 27 27" src="https://github.com/user-attachments/assets/4be1fe19-bb8b-44f6-96fc-7ae65bb4be41" />

Reply:
Fixed in: [#be95add](https://github.com/elastic/kibana/pull/204625/commits/be95add2656a764140377888697127a5faa9a53d)
I think all places are covered, but I might be wrong (AD jobs table, DFA jobs table, datafeed counts flyout)